src/engraving/libmscore/measure.cpp

```diff
@@ -2729,13 +2729,19 @@ Measure* Measure::mmRestLast() const
 }
 
 //---------------------------------------------------------
-//   mmRest1
-//    return the multi measure rest this measure is covered
-//    by
+//   coveringMMRestOrThis
+//    if multi-measure rests are enabled,
+//        and this measure is covered by an MMRest,
+//            return that MMRest.
+//    otherwise, return the measure itself.
 //---------------------------------------------------------
 
-const Measure* Measure::mmRest1() const
+const Measure* Measure::coveringMMRestOrThis() const
```

Comment:
maybe better add `to` - `coveringToMMRestOrThis`
otherwise, it seems to be the method that implements the process

Reply:
No, this name means: "the MMRest that is covering this measure, or this measure itself"